src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -205,8 +205,9 @@ uiModules.get('apps/management')
         })
         .then((overwriteAll) => {
           const conflictedIndexPatterns = [];
```

Comment:
Can we add a comment above these to provide some context?

```javascript
// We'll collect all index patterns which {don't correspond to a saved object? are missing? I'm not sure},
// so we can ask the user to {resolve them? reassign them?}. We'll also collect all saved searches
// which {don't correspond exist?}, so we can ask the user to {resolve them? recreate them?}.
const conflictedIndexPatterns = [];
const conflictedSearchDocs = [];
```

Reply:
```js
// Collect all index patterns and saved searches referenced in other documents, like visualizations,
// but which have not been imported yet.
```

Reply:
I think the naming here is a tad confusing. It makes more sense to call a set of index-patterns that could be associated with a savedsearch `conflictedIndexPatterns`, but doesn't make sense for savedsearches. The searchDocs are not conflicted. They just don't exist yet, haven't been imported yet, and so visualizations that reference them can't be imported.

Ideas:
`referencedSavedSearches` --> the viz references it. that's all we can say for sure.
`notYetImportedSavedSearches` --> this assumes that they will be imported. it could be that someone fucked with the id and the savedsearch doesn't actually exist.
`referencedAndNotFoundSavedSearches` -- verbose but accurate?

Reply:
I added some comments, does that help clarify this?